packages/apex-node/src/logs/logService.ts

```diff
@@ -90,8 +105,62 @@ export class LogService {
     return response.records as LogRecord[];
   }
 
+  private async createTraceFlag(userId: string): Promise<void> {
+    const DebugLevelId = await this.getDebugLevelId(DEFAULT_DEBUG_LEVEL_NAME);
+    const traceFlagDate = dayjs();
```

Comment:
Is it necessary to introduce a dependency here? JavaScript `Date` probably suffice what we need here.

Reply:
Good point! If we use dayjs (or moment js, like the other library had), does that make date testing/stubbing easier? If we don't worry about testing with dates, or that doesn't make a difference, I can definitely remove the dependency and simplify this

Reply:
For mocking Dates, sinon has some utilities. https://sinonjs.org/releases/v1.17.6/fake-timers/